### DISCUSSION TITLE
Space saved?

### DISCUSSION THREAD

**Discussion by JohnHens** [remay/tmf-hudl#27] [Q&A] [answered]
Hi,

I was wondering, does this release any space for more apps to be installed?

Thanks,
John

**Comment by remay** [comment, marked as answer]
Hi John,

No this doesn't make any additional memory available to install apps.  Based on my current understanding, then technically it might be possible to move some memory from the system partition (which is write only, and only about half used) and add it to the userdata partition (which is where apps are installed to).

But the userdata is already all of the 16GB flash except for just over 1GB that is used for other storage (and system is most of that).  We might get 400-500MB back from slimming down system.  I wonder if it's worth it to go from 15GB to 15.5GB.

Is running out of flash to install apps a real problem?

I have issue #6 open to look at this.  It's possible that the area reserved to install apps is a sub-partition of userdata and not all of it, but it would need some work to look further.

**Reply by JohnHens** [reply]
Hi and thanks for the quick reply. It isn't an issue, I only just realised how much user storage is free on this thing and it's great!! It's way more than on my Kindle Fire which is loaded with apps that can't be uninstalled and eat into the user storage space significantly. I was just wondering if increased storage was one of the benfits of this flash. Of course it's worth it just to know that I can do a factory reset in future and not have to worry about being locked out of my own device.
